modules/nested-tabs/assets/scss/frontend.scss

```diff
@@ -229,25 +229,33 @@
 		}
 	}
 
-	.e-con,
 	.e-n-tabs-content {
-		border: 1px none var( --n-tabs-border-color );
-		border-width: var( --n-tabs-border-width );
+
+		:where( & > .e-con ) {
+			border-radius: var(--border-radius);
+			padding-right: var(--padding-right);
+			padding-left: var(--padding-left);
```

Comment:
We had a discussion with Udi about this whether padding-left/right sould be on `e-con` or `e-con-inner` level. 

I am curious to hear your opinion. 

I am not 100% sure what would be best.

Reply:
This got to me thinking. I don't actually have to set any of these `.e-con` styles in our css.
This all gets handled by the Container css. All I need to do is stick to setting the vars and the CSS from the controls.

I've removed all that related css. And I will need to open a task on Monday to do the same the Mega Menu.
It's unnecessarily redundant code.